### ISSUE TITLE
Expose more outputs through the `Generator` interface

### ISSUE THREAD

**Issue by danieldk** [explosion/curated-transformers#345]
Expose more useful outputs, such as logits through the `Generator` interface. Also fixes #311 .
